core/modules/user/user.theme.inc

```diff
@@ -194,7 +204,9 @@ function theme_user_admin_roles($variables) {
 
   backdrop_add_tabledrag('user-roles', 'order', 'sibling', 'role-weight');
 
-  $output = theme('table', array('header' => $header, 'rows' => $rows, 'attributes' => array('id' => 'user-roles')));
+  $output = backdrop_render($form['roles_help_before']);
+  $output .= theme('table', array('header' => $header, 'rows' => $rows, 'attributes' => array('id' => 'user-roles')));
+  $output .= backdrop_render($form['roles_help_after']);
```

Comment:
As long as we get the correct `#weight` for this element, we don't need to add it separately. The only reason the `before` help was separate is because the table goes between it and the rest of the form.

Reply:
I may need some help with this. Before I even implemented things this way, I've tried setting the weight of the help text to something insane, like `'#weight' => -99999999,`, but the highest I could get it to render in the page was before the "Administrator role" fieldset 😓

Reply:
> but the highest I could get it to render in the page was before the "Administrator role" fieldset

That's where we want it, isn't it? That's where it is now:

<img width="1209" alt="Screen Shot 2020-07-27 at 4 31 21 PM" src="https://user-images.githubusercontent.com/397895/88867226-ba237280-d1c1-11ea-94f7-7b37cbc8fe3e.png">

Reply:
Oh, I see what you men now; I misread your initial comment and thought that you were suggesting to remove the `backdrop_render()` for `roles_help_before` as well.

Fixed 👍